app/models/utils/load_testing/odk_submission_load_test.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+module Utils
+  module LoadTesting
+    # Builds a JMeter test plan for submitting a lot of ODK form submissions.
+    #
+    # This test building process should be run on the server that is being targeted
+    # for the test because it will query the database while building the sample submission
+    # data (i.e. the XML for the ODK submission) for the test.
+    #
+    # The actual JMeter test will be run on a different server that does not have NEMO installed.
+    # It wil just know how to make a bunch of HTTP requests based on the options and data we give it.
+    class OdkSubmissionLoadTest < LoadTest
+      # Required `options`:
+      #   username: User doing the submitting
+      #   password: User's password
+      #   form_id: The ID of the form being submitted to.
+      #   mission_name: The name of the mission in which the form is being submitted
```

Comment:
I think we can get the mission from the form. Can you do so and remove this requirement?